### DISCUSSION TITLE
Cant connect to BungeeCord Proxy

### DISCUSSION THREAD

**Discussion by JuniorInjects** [Minestom/Minestom#1948] [Help] [answered]
![image](https://github.com/Minestom/Minestom/assets/87935854/2baeba13-19b0-4bc4-8256-d8402714be9a)

Can anyone help me with this error?

**Comment by mworzala** [comment, marked as answer]
Use `BungeeCordProxy.enable()`
If using BungeeGuard, also call `BungeeCordProxy.setBungeeGuardTokens(Set.of("token here"))`

**Reply by JuniorInjects** [reply]
i already have

**Reply by mworzala** [reply]
Are you using bungeeguard? There are some extra options which need to be provided in `BungeeCordProxy` if so.

**Reply by JuniorInjects** [reply]
No, how can is enable bungeeguard?

**Reply by mworzala** [reply]
Use `BungeeCordProxy.setBungeeGuardTokens(Set.of("token here"))`

**Reply by JuniorInjects** [reply]
where can i find the token?

**Reply by mworzala** [reply]
I have never used bungeeguard, you will have to refer to their docs/ask them
